src/components/chart/line-chart.js

```diff
@@ -0,0 +1,19 @@
+class LineChartPlotOptions {
+    static plotOptions() {
+        return this.plotOptions;
+    }
+
+    constructor() {
+        this.plotOptions = {
+            line: {
+                lineWidth: 3,
+                linecap: 'round',
+                marker: {
+                    enabled: false,
```

Comment:
This needs to be true if markers have been enabled. We then also don't need to update the last point marker to display.

Reply:
Discussed on our call. The marker parameter is optional and may be present or not. If it isn't present, it will default to false. If it is present, and is also `true`, then we need to update this line, and also make sure that we don't run the code in the load events that hides all the markers except the last one.